tests/integration/targets/ios_vrf_address_family/tests/common/parsed.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- ansible.builtin.debug:
+    msg: START ios_vrf_address_family parsed integration tests on connection={{ ansible_connection }}
+
+- name: Parse externally provided VRF configuration
+  register: result
+  cisco.ios.ios_vrf_address_family:
+    running_config: "{{ lookup('file', './_parsed.cfg') }}"
+    state: parsed
+
+- ansible.builtin.assert:
+    that:
+      - result.changed == false
+      # - parsed['after'] == result['parsed']
```

Comment:
I believe its not supposed tobe  commented

Reply:
removed, added it by mistakely!